solr/core/src/java/org/apache/solr/cloud/SolrZkServer.java

```diff
@@ -70,7 +70,7 @@ public String getClientString() {
 
     // if the string wasn't passed as zkHost, then use the standalone server we started
     if (zkRun == null) return null;
-    return "localhost:" + zkProps.getClientPortAddress().getPort();
+    return zkProps.getClientPortAddress().getHostString() + ":" + zkProps.getClientPortAddress().getPort();
```

Comment:
LGTM. I was trying to see if we need to switch to using zkProps.getSecureClientPortAddress optionally but couldn't find anything in the ZK javadocs. Also, based on my ZK TLS testing - didn't seem like that was needed.